### ISSUE TITLE
foxnews.com won't load

### ISSUE THREAD

**Issue by jacobsalmela** [pi-hole/pi-hole#382]
##### Expected Behaviour:
The Website should load on iOS and OS X.

##### Actual Behaviour:
A blank page is loaded as well as a script.  Even after whitelisting the following domains:
```
www.foxnews.com
a57.foxnews.com
global.fncstatic.com
foxnews.demdex.net
cdn.tt.omtrdc.net
foxnews.tt.omtrdc.net
hpr.outbrain.com
d5i9o0tpq9sa1.cloudfront.net
assets.tapad.com
a.visualrevenue.com
metrics.foxnews.com
mpp.vindicosuite.com
tapestry.tapad.com
widgets.outbrain.com
images.outbrain.com
log.outbrain.com
cdn.livefyre.com
widget-cdn.rpxnow.com
d1v9u0bgi1uimx.cloudfront.net
cdn.quilt.janrain.com
bootstrap.livefyre.com
d134l0cdryxgwa.cloudfront.net
d1lqe9temigv1p.cloudfront.net
fonts.gstatic.com
sso.foxnews.com
fast.foxnews.demdex.net
s7.addthis.com
g.symcd.com
s2.symcb.com
```

##### Steps to reproduce this issue:
Existing install with default lists, foxnews.com will not load.

**Comment by manuelgu** [comment]
Cannot reproduce. The page loads fine for me even without whitelisting any additional domains.

**Comment by dschaper** [comment]
This seems to be limited to Mac OS platforms. Are you running on an iDevice?

**Comment by manuelgu** [comment]
Can confirm what @dschaper said. It is working fine on Google Chrome on Linux Mint and Windows, not on an iPad though.

**Comment by bukka1** [comment]
Loading fine here with Safari on OS X 10.11.3

**Comment by jacobsalmela** [comment]
Sorry I meant to include that it seems to be just OS X and iOS that it doesn't work on.

**Comment by kmr2319** [comment]
@jacobsalmela  So you're going to fix it so it's blocked on all platforms? :wink: 

Sorry... I had to. It was just too easy. :smiley:

**Comment by jacobsalmela** [comment]
Haha.  That's the plan!  

@bukka1 could you try emptying your browser cache and DNS cache from within OS X: `sudo dscacheutil -flushcache; sudo killall -HUP mDNSResponder` and then try again?  Just to rule out any caching issues?

**Comment by colemannugent** [comment]
I've had the exact same issue with foxnews.com. Whitelisting fixed it for a while, but every couple days it will be blocked again. I've been re-running `gravity.sh` and restarting dnsmasq with `sudo service dnsmasq restart` and that seems to fix this at least temporarily.

This seems to be independent of OS caching, as I've had this issue on Windows (7 & 10), and Linux (Ubuntu 15.10)

**Comment by dschaper** [comment]
Okay, I got a RasPi 2B on the client network, I'll set if I can figure out Mac networking well enough to see if I can run a few tests when I'm back on site in a few...

**Comment by bukka1** [comment]
@jacobsalmela flushed, still working for me.

**Comment by Mcat12** [comment]
Still an issue?

**Comment by colemannugent** [comment]
I haven't had any issues lately, this may be fixed. I have a feeling it had to do with something about the way the whitelists were handled, but I have nothing to prove it.

**Comment by azleatherneck** [comment]
foxnews hasnt been loading for 3 days have tried the normal things, whitelist, update gravity restart dnsmasq etc just goes to the cant resolve dns not a blank page like an ad. Running Pihole on another network same version same setup but it can get to foxnews "most" of the time. Disable Pihole no issue. Kinda stumped here.

**Comment by dschaper** [comment]
Do both of the Pi-Hole setups use the same upstream DNS provider as their sources? Since the Pi-Hole uses `dnsmasq` to either return the DNS query information from it's internal host file that we create or if the domain is not found there, then it queries the upstream DNS provider that is selected during setup.

Typically we don't see failures in the resolving part of the configuration, if the once Pi-Hole is repeatedly failing and returning the 'Cant resolve' error then we may need to take a look at the Pi-Hole resolver logging while you are trying to view FoxNews and troubleshoot from there. 

Since this appears to be unrelated to the Whitelist/Blacklist error that this thread covers, would it be okay if you opened a new issue under a title like `FoxNews fails to resolve` so we can track the issue better? (This thread has been closed and I'd like to get you an open thread so we can troubleshoot your issue and not get you lost in the depths of closed issues!)

**Comment by azleatherneck** [comment]
Different upstream servers I'll test that out first. Sure move away and thank you for the response I'll advise on results of changing upstream servers

**Comment by dschaper** [comment]
Yeah, this is a different failure mode than I have seen so I'd like to see this through and discover where the issue is. I'm unable to move this to a new topic, so I'd need you to open a new issue when you get the chance.

**Comment by azleatherneck** [comment]
open new issue

**Comment by dschaper** [comment]
From the main Issues page, (with all the issues listed) in the upper left hand corner should be a green button for New Issue. Just click on that and there will be a little template that shows up to describe the problem. You can be a little brief on that, we know the description of the issue.

**Comment by iannao** [comment]
Hi All,
did you block the call to fast.foxnews.demdex.net? you can use fiddler in order to block that call.

Please let me know if this resolved the issue

Tahnk you
Best Regards
Riccardo

**Comment by dschaper** [comment]
Thanks, the issue was resolved in a later release. Since this is no longer a problem, I'm going to lock out the thread. If any more problems come up a new issues needs to be raised.
